content/docs/multi-tenancy/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Multi-Tenancy in Kubeflow"
+description = "Documentation for Multi-Tenancy and IAM in Kubeflow"
```

Comment:
Our style is to use title case (capital letters on most words) for page titles, but not for descriptions or for headings within a page. So, please would you change this line to sentence case (see suggestion below).

Also, we should expand the abbreviation "IAM" as many readers won't know what it means. And we should use a different term from "multi-tenancy", because people can see that in the title but may not know what it means.

I think we can leave out the words "documentation" and "kubeflow".

Something like this:

```
Multi-user isolation and identity access management (IAM)
```